src/templates/hi/2019/methodology.html

```diff
@@ -0,0 +1,428 @@
+{% extends "base/methodology.html" %}
+
+{% block title %}कार्यप्रणाली | HTTP Archive द्वारा Web Almanac{% endblock %}
+
+{% block description %}यह बताता है कि 2019 Web Almanac को कैसे एक साथ रखा गया था: किस तरह से डेटासेट्स और टूल्स का उपयोग किया गया और इस परियोजना को शुरु किया गया था।{% endblock %}
+
+{% block twitter_image_alt %}{{ year }} Web Almanac की कार्यप्रणाली{% endblock %}
+
+
```

Comment:
Remove extra line to keep line numbers consistent with English

```suggestion
```

Reply:
Done